platform/android/detect.py

```diff
@@ -184,13 +184,16 @@ def configure(env: "SConsEnvironment"):
     if env["arch"] == "x86_32":
         # The NDK adds this if targeting API < 24, so we can drop it when Godot targets it at least
         env.Append(CCFLAGS=["-mstackrealign"])
+        env.Append(LIBPATH=["../../thirdparty/swappy-frame-pacing/x86"])
```

Comment:
Binary libs in the tree? This should be either build in tree or optional external dependency.